tests/test_tokenization_mistral_common.py

```diff
@@ -486,6 +486,11 @@ def test_convert_tokens_to_ids(self):
         ids = self.tokenizer.convert_tokens_to_ids([])
         self.assertEqual(ids, [])
 
+    def test_convert_tokens_to_string(self):
+        tokens = ["Hello", "world", "!"]
+        string = self.tokenizer.convert_tokens_to_string(tokens)
+        self.assertIsInstance(string, str)
```

Comment:
can you do a bit more testing! like what do you expect as string here, tokenizer -> convert tokens to string